### PR TITLE
Added Onet and cda

### PR DESCRIPTION
**Pull request by wkuba500**

### CODE REVIEW

File: sites.json [hunk 1]
```diff
@@ -862,6 +862,16 @@
     },
 
     {
+        "name": "cda",
+        "url": "https://www.cda.pl/kontakt",
+        "difficulty": "hard",
+        "notes": "You need to contact with support",
```
Comment (Nightfirecat) [inline sites.json]: Does this need to be done via email?

File: sites.json [hunk 2]
```diff
@@ -862,6 +862,16 @@
...
+        "domains": [
+            "cda.pl"
+        ]
+    },
+
```
Comment (Nightfirecat) [inline sites.json]: Please remove this trailing space.

File: sites.json [hunk 3]
```diff
@@ -4319,6 +4329,16 @@
...
     {
+        "name": "Onet",
+        "url": "https://konto.onet.pl/data.html",
+        "difficulty": "easy",
+        "notes": "You just need to click 'Delete account', then enter your account password and tick the checkbox",
+        "domains": [
+            "onet.pl"
+        ]
+    },
+
```
Comment (Nightfirecat) [inline sites.json]: Please remove this trailing space.

**Comment by Nightfirecat** [comment]
Thank you for your contribution!

File: sites.json (hunk 1 above)
Comment (wkuba500) [reply sites.json]: Unfortunately yes
There is an email at the bottom of https://www.cda.pl/kontakt

**Pull request merged**